Apps/W1/EDocumentsConnector/app/src/Logiq/LogiqConnectionUserSetup.Table.al

```diff
@@ -0,0 +1,146 @@
+namespace JLogiqEDocumentsConnector.JLogiqEDocumentsConnector;
+
+using System.Security.AccessControl;
+
+table 6381 "Logiq Connection User Setup"
+{
+    Caption = 'Logiq Connection User Setup';
+    DataClassification = CustomerContent;
+
+    fields
+    {
+        field(1; "User ID"; Text[50])
```

Comment:
a Field that contains the User ID / User Name should use the correct DataClassification.
`DataClassification = EndUserIdentifiableInformation;`

Reply:
Added DataClassification = EndUserIdentifiableInformation to "User ID" field